From 6fb58d4e83fcb4fc453990d6367c61ae6f4222e8 Mon Sep 17 00:00:00 2001
From: md_5 <md_5@live.com.au>
Date: Thu, 4 Jul 2013 20:05:19 +1000
Subject: [PATCH] Update Depends

- All of these changes have been reviewed to be binary compatible and in general contract compatible with previous versions of the libraries.

diff --git a/pom.xml b/pom.xml
index c5ce4b2..b5ad32f 100644
--- a/pom.xml
+++ b/pom.xml
@@ -83,21 +83,21 @@
         <dependency>
             <groupId>org.yaml</groupId>
             <artifactId>snakeyaml</artifactId>
-            <version>1.9</version>
+            <version>1.12</version>
             <type>jar</type>
             <scope>compile</scope>
         </dependency>
         <dependency>
             <groupId>com.googlecode.json-simple</groupId>
             <artifactId>json-simple</artifactId>
-            <version>1.1</version>
+            <version>1.1.1</version>
             <type>jar</type>
             <scope>compile</scope>
         </dependency>
         <dependency>
             <groupId>org.avaje</groupId>
             <artifactId>ebean</artifactId>
-            <version>2.7.3</version>
+            <version>2.8.1</version>
             <type>jar</type>
             <scope>compile</scope>
         </dependency>
@@ -112,7 +112,7 @@
         <dependency>
             <groupId>commons-lang</groupId>
             <artifactId>commons-lang</artifactId>
-            <version>2.3</version>
+            <version>2.6</version>
         </dependency>
 
         <!-- testing -->
-- 
2.1.0

